Web3/Classes/Core/Transaction/EthereumPublicKey.swift

```diff
@@ -32,7 +32,7 @@ public final class EthereumPublicKey {
     /**
      * Convenient initializer for `init(publicKey:)`
      */
-    public required convenience init(bytes: Bytes) throws {
+    public required convenience init(_ bytes: Bytes) throws {
```

Comment:
Why?

Reply:
Same question for all initializers you changed...

Reply:
A new style in Swift 4.2. For Foundation types, they have `init(_ bytes: Sequence)` initializers. It was `init(bytes: Sequence)` before. If we do it this way, we can add BytesInitializable for some of the Foundation classes automatically. And it doesn't show warnings for them (because other way it will show 'deprecated' warning)

Reply:
I guess it's fine then. What do you think @pixelmatrix ?

And what about the initializer where you changed `bytes` to `raw`?

Reply:
It was an attempt to fix warnings without `init(_)`. Then I understood that it's better to use Swift way. If they updated it to `init(_)` we should do the same

Reply:
Makes sense to me to try to align with Foundation's initializers